.ci/macos-steps.yaml

```
@@ -22,5 +22,9 @@ steps:
   displayName: 'CMake'
 
 # Build mlpack
-- script: cd build && make 
-  displayName: 'Build'
\ No newline at end of file
+- script: cd build/include && ln -s /usr/local/include/google ./google && cd ../ && make 
+  displayName: 'Build'
+
+# Run test
+- script: ./mlboard_tests   
```

I'm not an azure pipelines wizard, but I think pwd may reset after each step.
```suggestion
- script: cd build && ./mlboard_tests
```